src/frontend/Komponenter/Personoversikt/Dokumenter.tsx

```diff
@@ -54,6 +70,50 @@ const InnUt = styled.div`
     }
 `;
 
+const ArkivtemaVelger = styled(FamilieReactSelect)`
+    width: 25rem;
+    margin: 0.5rem 1rem 0 0;
+
+    .react-select__control {
+        min-height: 3rem;
+    }
+`;
+
+const Dokumenttabell = styled.div`
```

Comment:
Til info: Denne er i stor grad en kopi av `TabellWrapper`, men vi trengte noe egen styling for å få filtreringen til å se ok ut.